.github/workflows/ci.yaml

```diff
@@ -2,13 +2,7 @@ name: CI
 
 on:
   push:
-    branches:
-      - main
```

Comment:
lets run on push only for main, develop and feature_*

Reply:
ok